.github/workflows/ci.yml

```
@@ -206,7 +206,7 @@ jobs:
     strategy:
       matrix:
         platform: [amd64, arm64]
-        python_version: ['3.9', '3.11']
+        python_version: ['3.8', '3.11']
```

Any details on this change? I suspect this is the reason the CI says that not all the tests are passing.

---

Could we / should we change this back to 3.9, at least for the purposes of the merge?

---

Suggest reverting this back to what is on main right now, and we will sort out the rest post-merge.
```suggestion
        python_version: ['3.9', '3.11']
```